scripts/views/eventView.js

```diff
@@ -9,10 +9,30 @@
   eventHandler.lookup = function (e) {
     e.preventDefault();
     var zip = $('#look-up input').val();
-    if (zip) {
-      TownHall.lookupZip($('#look-up input').val());
-      eventHandler.resetFilters();
+    if (zip && zip.length === 5) {
```

Comment:
For the entire if{} elseif{} that handles parsing the zip, is it worth having a catch-all else{} also?  A lot of garbage input can still get through and we make the API call.